src/init/AppRunner.ts

```diff
@@ -56,97 +50,77 @@ export class AppRunner {
   } = {}): void {
     // Parse the command-line arguments
     // eslint-disable-next-line no-sync
-    const params = yargs(argv.slice(2))
-      .strict()
-      .usage('node ./bin/server.js [args]')
-      .check((args): boolean => {
-        if (args._.length > 0) {
-          throw new Error(`Unsupported positional arguments: "${args._.join('", "')}"`);
-        }
-        for (const key of Object.keys(args)) {
-          // We have no options that allow for arrays
-          const val = args[key];
-          if (key !== '_' && Array.isArray(val)) {
-            throw new Error(`Multiple values were provided for: "${key}": "${val.join('", "')}"`);
-          }
-        }
-        return true;
-      })
-      .options({
-        baseUrl: { type: 'string', alias: 'b', requiresArg: true },
-        config: { type: 'string', alias: 'c', default: defaultConfig, requiresArg: true },
-        loggingLevel: { type: 'string', alias: 'l', default: 'info', requiresArg: true },
-        mainModulePath: { type: 'string', alias: 'm', requiresArg: true },
-        port: { type: 'number', alias: 'p', default: 3000, requiresArg: true },
-        rootFilePath: { type: 'string', alias: 'f', default: './', requiresArg: true },
-        showStackTrace: { type: 'boolean', alias: 't', default: false },
-        sparqlEndpoint: { type: 'string', alias: 's', requiresArg: true },
-        podConfigJson: { type: 'string', default: './pod-config.json', requiresArg: true },
-      })
-      .parseSync();
+    const params = setupYargvWithBaseArgs(yargs(argv.slice(2))
+      .usage('node ./bin/server.js [args]')).parseSync();
 
-    // Gather settings for instantiating the server
-    const loaderProperties: IComponentsManagerBuilderOptions<App> = {
+    // Gather settings for instantiating VarResolver
+    const loaderProperties = {
       mainModulePath: resolveAssetPath(params.mainModulePath),
       dumpErrorState: true,
       logLevel: params.loggingLevel as LogLevel,
     };
-    const configFile = resolveAssetPath(params.config);
 
-    // Create and execute the app
-    this.createApp(loaderProperties, configFile, params)
-      .then(
-        async(app): Promise<void> => app.start(),
-        (error: Error): void => {
-          // Instantiation of components has failed, so there is no logger to use
-          stderr.write(`Error: could not instantiate server from ${configFile}\n`);
-          stderr.write(`${error.stack}\n`);
-          process.exit(1);
-        },
-      ).catch((error): void => {
-        this.logger.error(`Could not start server: ${error}`, { error });
-        process.exit(1);
-      });
+    // Create varResolver
+    const varConfigFile = resolveAssetPath(params.varConfig);
```

Comment:
```suggestion
    const cliConfig = resolveAssetPath(params.cliConfig);
```

Reply:
Here configuration about componentsjs variables. cli args are part of configuration, but other configurations like computeHandlers, etc. are part of how variables should be resolved